aiidalab_widgets_base/structures.py

```diff
@@ -114,7 +114,19 @@ def __init__(
                 f"Unknown data format '{node_class}'. Options: {list(self.SUPPORTED_DATA_FORMATS.keys())}"
             )
 
-        viewer_panel = ipw.Accordion(
+        structure_importers = self._structure_importers(importers)
+
+        select_panel = ipw.Accordion(
+            children=[
+                *structure_importers,
+            ]
+            if structure_importers
+            else [],
+            selected_index=0 if structure_importers else None,
+        )
```

Comment:
I think the logic here has already been taken care of by the `_structure_importers()`, so one can just do:
 ```suggestion
        select_panel = ipw.Accordion(
            children=self._structure_importers(importers)
        )
```

Reply:
Well, if no importers, you get back a Tab with no children, which is still a child, no? I mean, I guess this is okay. Technically if developers don't provide any children, then what are they even doing?

Reply:
if no importers, you get an empty list as far as I can see:

```python
        if not importers:
            return []
```

Reply:
My point is that this lifts the logic up to the accordion, so no importers now mean an empty accordion, not an accordion with an empty Tab. Which do we prefer? Again, either case is nonesense from a development point of view.

Reply:
Oh right, my bad. Missed that if statement. Okay then 👍

Reply:
Keeping the `selected_index` conditional though, yeah?
```python
        select_panel = ipw.Accordion(
            children=self._structure_importers(importers),
            selected_index=0 if importers else None,
        )
        select_panel.set_title(0, "Select structure")
```

Reply:
You can, but this is not necessary. You are literally replicating the default behaviour.